packages/republik-crowdfundings/lib/scheduler/owners.js

```diff
@@ -291,3 +294,34 @@ module.exports = {
   DAYS_BEFORE_END_DATE,
   run
 }
+
+async function autoPayMailings (user, bucket, context) {
+  const pgdb = context.pgdb
+  const defaultPaymentSource = getDefaultPaymentSource(user, pgdb)
+
+  if (!defaultPaymentSource) {
+    await setAutoPayToFalse(user, pgdb)
+    return
+  }
+
+  user = { ...user, defaultPaymentSource }
+  return mailings(user, bucket, context)
```

Comment:
(card is expected on `user.autoPay.card` by downstream functions not `user.defaultPaymentSource`, but no longer relevant if we move to `suggest`)